resources/postgres/postgres.go

```diff
@@ -34,20 +34,29 @@ func (i Instance) Port() int { return i.port }
 
 // Start starts a postgres instance on given port, outputs to stdout and stderr.
 //
+// If host == "", postgres will only listen the Unix-domain socket, else it will only listen the host
+//
 // If port is 0, then it will choose a random unused port.
 //
 // If waitSec > 0, watis at most `waitSec` seconds for the postgres instance to start.
 // Otherwise, returns immediately.
-func (i *Instance) Start(port int, stdout, stderr io.Writer) (err error) {
+func (i *Instance) Start(host string, port int, stdout, stderr io.Writer) (err error) {
```

Comment:
I would still prefer only listening to unix domain socket only without giving an option of listening on host. We can spend a bit more time to figure out why unix socket connection doesn't work for action tests.

Since we were able to connect to embedded postgres using unix socket, I'm sure we can do so via external postgres.

Reply:
Yes, I was just trying to listen to the host to see if the GitHub action test would work. Unfortunately, neither listening on the host nor the Unix domain socket is feasible.